src/Radio/index.scss

```diff
@@ -0,0 +1,47 @@
+.nds-singleRadio {
+  position: relative;
+}
+
+.nds-singleRadio input {
+  position: absolute;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
```

Comment:
This should probably be a utility class, but that's out of scope for this PR.

Basically this set of rules hides the actual `input` element from users who use their eyes to find the element, but allows it to remain visible for assistive tech.

If you set `display` to `none`, it will be hidden from assistive tech.

This is also why I place a `presentation` role on the div that paints the faux input.